client/src/App.test.js

```diff
@@ -1,9 +1,52 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import * as rtl from "@testing-library/react";
+// import "@testing-library/jest-dom/extend-expect";
+import App from "./App";
+import { isMainThread } from 'worker_threads';
+import Display from "./components/Display";
+import PlayerCard from "./components/PlayerCard";
+import useDarkMode from "./components/hooks/useDarkMode";
+import useLocalStorage from "./components/hooks/useLocalStorage"
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+
+test('renders PlayerCard component', () => {
+  const wrapper = rtl.render(<PlayerCard />);
+  console.log(wrapper.debug());
+});
+
+
+test('renders localstorage', () => {
```

Comment:
useLocalStorage is a hook not a React component so it can't be rendered. You'll need to use different tests to check your hooks.